api/local_analytics_exporter.py

```diff
@@ -43,8 +43,7 @@ def export(self, _db, start, end, locations=None, library=None):
         writer = csv.writer(output, encoding="utf-8")
         writer.writerow(header)
         writer.writerows(results)
-
-        return output.getvalue()
+        return output.getvalue().decode("utf-8")
```

Comment:
I was hoping we could avoid this encode/decode step but it looks like we can't.

Reply:
Yeah, not for BytesIO buffer.